x-pack/plugins/endpoint/common/generate_data.test.ts

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { EndpointDocGenerator, generateEventAncestry, generateResolverTree } from './generate_data';
```

Comment:
Why aren't `generateEventAncestry` and `generateResolverTree` methods on the `EndpointDocGenerator` class? It seems the generator is being passed into the functions anyway...

Reply:
true, done